app/config/FrontendAppConfig.scala

```diff
@@ -78,4 +81,12 @@ class FrontendAppConfig @Inject()(override val runModeConfiguration: Configurati
   lazy val publishedAssets = loadConfig(s"urls.external.assets.host")
   def getPublishedAssetsUrl(key: String): String = publishedAssets + loadConfig(s"urls.external.assets.$key")
 
+  def emacEnrollmentsUrl(enrolment: Enrolments): String = {
+    s"$enrolmentManagementFrontendHost/enrolment-management-frontend/${enrolment.toString}/request-access-tax-scheme?continue=%2Fbusiness-account"
```

Comment:
Should we put this in the config?

Reply:
yes, we should - changing. I just lifted as it is.